### ISSUE TITLE
STA9750 Timbila614 MiniProject #00

### ISSUE THREAD

**Issue by Timbila614** [Timbila614/STA9750-2024-FALL#1]
Hi @michaelweylandt!

I've created my STA9750 website - check it out!

https://Timbila614.github.io/STA9750-2024-FALL

**Comment by michaelweylandt** [comment]
Thanks @Timbila614 !

Peer Feedback:

- @aaronkcooper
- @matata668
- @mt1536

Please a look at the website associated with this GitHub repo
and provide some peer feedback to @Timbila614.

Website URL should be: https://Timbila614.github.io/STA9750-2024-FALL/

If that link doesn't work, please check out the GitHub profile for @Timbila614 and
see if you can help them get it working.

At a minimum, please note:

1. Something you like about the submission.
2. A suggestion on how to improve it.

Feel free to link to other repos, the course documentation, or other useful
examples.

This submission is ungraded, so there's you don't need to provide
a numerical score.

Thanks!
@michaelweylandt

CC: @charles-ramirez

**Comment by mt1536** [comment]
Hi @Timbila614,

I like how clean and organized your website looks. Currently I have no suggestions on how to improve it.

Regards,
mt1536

**Comment by aaronkcooper** [comment]
Hi, 
I like that the website is clean and is running well. I would suggest to try to break up the paragraphs as it is a bit wordy not that great to look at. Maybe a better way of breaking down your experiences/bio, adding pictures or color too may help.
